css-paint-api/Overview.bs

```diff
@@ -131,10 +131,16 @@ partial interface CSS {
 
 The {{PaintWorkletGlobalScope}} is the global execution context of the {{paintWorklet}}.
 
+The {{PaintWorkletGlobalScope}} has a {{PaintWorkletGlobalScope/devicePixelRatio}} property,
+which represents the ratio of the resolution in physical pixels to the resolution of CSS
+pixels for the current display device. When measured from the same display device, this
+value must be identical to the Window.{{Window/devicePixelRatio}}.
```

Comment:
Since we link to the spec which has a more precise definition of the device pixel ratio it probably makes sense to shorten this and not try to redefine what it means here. i.e. How about just The PaintWorkletGlobalScope has a devicePixelRatio property which is identical to the Window.devicePixelRatio property.